lib/mayaUsd/ufe/UsdAttributes.h

```diff
@@ -23,7 +23,7 @@
 
 #include <ufe/attributes.h>
 #ifdef UFE_V4_FEATURES_AVAILABLE
-#if (UFE_PREVIEW_VERSION_NUM >= 4008)
+#if (UFE_PREVIEW_VERSION_NUM >= 4010)
```

Comment:
The CMakefile all use 4010, so make sure we use a unique Ufe version. There was never a release using 4008 or 4009.